test/ui-automation/test/specs/vault-interaction-flow.js

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+"use strict";
+
+import {wallet} from "../helpers";
+
+const vault = new Map();
+vault.set("Test Vault 1", {
+    "@context":["https://w3id.org/wallet/v1",
+        "https://trustbloc.github.io/context/wallet/collections-v1.jsonld"],
+    "id":"1",
+    "name":"Test Vault 1",
+    "type":"Vault"
+});
+vault.set("Test Vault 2", {
+    "@context":["https://w3id.org/wallet/v1",
+        "https://trustbloc.github.io/context/wallet/collections-v1.jsonld"],
+    "id":"2",
+    "name":"Test Vault 2",
+    "type":"Vault"
+});
+vault.set("Test Vault 3", {
+    "@context":["https://w3id.org/wallet/v1",
+        "https://trustbloc.github.io/context/wallet/collections-v1.jsonld"],
+    "id":"3",
+    "name":"Test Vault 3",
+    "type":"Vault"
+});
+vault.set("Test Vault 4", {
+    "@context":["https://w3id.org/wallet/v1",
+        "https://trustbloc.github.io/context/wallet/collections-v1.jsonld"],
+    "id":"4",
+    "name":"Test Vault 4",
+    "type":"Vault"
+});
+
+describe("TrustBloc Wallet - Add/Rename/Delete Vault flow", () => {
+    const ctx = {
+        email: `ui-aut-${new Date().getTime()}@test.com`,
+    };
+    const invalidVaultName ='Vault!@';
+    const validVaultName = "Test Vault 4";
+
+    // runs once before the first test in this block
+    before(async () => {
+        await browser.reloadSession();
+        await browser.maximizeWindow();
+    });
+
+    beforeEach(function () {});
+
+    it(`User Sign up (${ctx.email})`, async function () {
+        this.timeout(90000);
+
+        // 1. Navigate to Wallet Website
+        await browser.navigateTo(browser.config.walletURL);
+
+        // 2. Initialize Wallet (register/sign-up/etc.)
+        await wallet.init(ctx);
+
+        // 3. Go to Vaults screen
+        const vaultsLink = await $("#navbar-link-vaults");
```

Comment:
Do we need to explicitly go to vault screen ? Default screen after login already points to vault screen.